tests/vxingest/builder_common/test_unit_queries.py

```diff
@@ -43,6 +45,37 @@ def connect_cb():
     return cb_connection
 
 
+@pytest.mark.integration
+def test_get_file_list(request):
+    vx_ingest = VXIngest()
+    vx_ingest.credentials_file = os.environ["CREDENTIALS"]
+    vx_ingest.cb_credentials = vx_ingest.get_credentials(vx_ingest.load_spec)
+    vx_ingest.connect_cb()
+    testdata = Path("tests/vxingest/builder_common/testdata/get_file_list_grib2.n1ql")
+    with testdata.open(mode="r", encoding="utf-8") as file:
+        _statement = file.read()
+    temp_dir_path = "/tmp/vx_ingest"
+    shutil.rmtree(temp_dir_path, ignore_errors=True)
+    Path(temp_dir_path).mkdir(parents=True, exist_ok=True)
+    with Path(os.path.join(temp_dir_path, "2128723000010")).open("w") as f:
+        f.write("test")
+    with open(os.path.join(temp_dir_path, "2128723000020"), "w") as f:
+        f.write("test")
+    with open(os.path.join(temp_dir_path, "2128723000030"), "w") as f:
+        f.write("test")
+    with open(os.path.join(temp_dir_path, "2128723000040"), "w") as f:
+        f.write("test")
+
+    file_list = vx_ingest.get_file_list(
+        _statement, temp_dir_path, "21287230000[0123456789]?", "%y%j%H%f"
+    )
+    assert file_list is not None, f"{request.node.name}: file_list is None"
+    assert len(file_list) > 0, f"{request.node.name}: file_list is empty"
+    assert (
+        file_list[3] > file_list[2]
+    ), f"{request.node.name}: file_list is not reverse sorted"
```

Comment:
Pytest will drop the function name in the assert output if it comes up false so we shouldn't need to include `request.node.name` and can remove that & the fixture from the code.

Pytest will also have equivalent output for the first two error messages so those messages can be left out. However, the `file_list is not reverse sorted` message is useful for conveying the intent. (Without it, pytest will say something like `test_get_file_list(): 5 < 3` in that case)